src/d3.timeslider.coffee

```diff
@@ -225,12 +228,53 @@ class TimeSlider extends EventEmitter
                 .attr('y', 0)
 
         # add a group to draw the brush in
-        @svg.append('g')
+        @gBrush = @svg.append('g')
             .attr('class', 'brush')
             .call(@brush)
-            .selectAll('rect')
+
+        if !@options.alternativeBrush
+            # traditional view with box for selection
+            @gBrush.selectAll('rect')
                 .attr('height', "#{@options.height - 19}")
                 .attr('y', 0)
+        else
+        # alternative brush
+            @gBrush.selectAll('rect')
+                .attr('height', 6)
+                .attr('y', "#{@options.height + 6}")
+            @handleShape = d3.svg.arc()
```

Comment:
Is there a particular reason to use arc instead of a simple `circle` element? This looks a little complicated.